### PR TITLE
chore: Update data pipeline blips

### PR DESCRIPTION
**Pull request by andrablaj**
Move CHT Sync and CHT Pipeline to `Adopt`.

### CODE REVIEW

File: radar/2024-09-04/couch2pg.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ring:       stop
+quadrant:   methods-and-patterns
+tags:       [data-use]
+---
+
+[couch2pg](https://github.com/medic/couch2pg) streams data from CouchDB and forwards it to PostgreSQL.
+
+An updated version of couch2pg is [embedded in CHT Sync repository](https://github.com/medic/cht-sync/tree/main/couch2pg) and no longer uses the [deprecated repository](https://github.com/medic/couch2pg). 
+
+For data synchronization, refer to [CHT Sync documentation](https://docs.communityhealthtoolkit.org/core/overview/cht-sync/).
```
Comment (garethbowen) [inline radar/2024-09-04/couch2pg.md:12]: The naming here is really confusing. I think we should drop the term "couch2pg" altogether - it's fine if it lives on internally, but externally there is no such thing.

```suggestion
couch2pg is deprecated. For data synchronization, refer to [CHT Sync documentation](https://docs.communityhealthtoolkit.org/core/overview/cht-sync/).
```

File: radar/2024-09-04/cht-couch2pg.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title:      "CHT couch2pg"
+ring:       stop
+quadrant:   methods-and-patterns
+tags:       [data-use]
+---
+
+CHT chouch2pg is deprecated. For data synchronization, refer to [CHT Sync documentation](https://docs.communityhealthtoolkit.org/core/overview/cht-sync/).
```
Comment (garethbowen) [inline radar/2024-09-04/cht-couch2pg.md:8]:
```suggestion
CHT couch2pg is deprecated. For data synchronization, refer to [CHT Sync documentation](https://docs.communityhealthtoolkit.org/core/overview/cht-sync/).
```

File: radar/2024-09-04/cht-sync.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title:      "CHT Sync"
+ring:       adopt
+quadrant:   methods-and-patterns
+tags:       [data-use]
+---
+
+[CHT Sync](https://github.com/medic/cht-sync) is a integrated solution designed to enable data synchronization between CouchDB and PostgreSQL for the purpose of analytics. The synchronization occurs in real-time, ensuring that the data displayed on dashboards is up-to-date. It combines several technologies to achieve this synchronization (an updated version of [couch2pg](https://github.com/medic/cht-sync/tree/main/couch2pg), CouchDB, PostgreSQL, dbt) and provides an efficient workflow for data processing and visualization.
```
Comment (garethbowen) [inline radar/2024-09-04/cht-sync.md:8]: "in real-time" is probably overselling it a bit.

Also let's drop couch2pg - it's  an internal detail and not relevant to implementers.

```suggestion
[CHT Sync](https://github.com/medic/cht-sync) is a integrated solution designed to enable data synchronization between CouchDB and PostgreSQL for the purpose of analytics. The synchronization occurs near real-time, ensuring that the data displayed on dashboards is up-to-date. It combines several technologies to achieve this synchronization and provides an efficient workflow for data processing and visualization.
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Thanks!

File: radar/2024-09-04/cht-couch2pg.md (hunk 2 above)
Comment (andrablaj) [reply radar/2024-09-04/cht-couch2pg.md:8]: Oups! 😅

**Review by garethbowen — APPROVED** [review]
Approved to unblock.

There's an outstanding discussion that needs to be resolved around what value there is in documenting cht-sync and cht-pipeline separately: https://medic.slack.com/archives/C03SP4KRVEX/p1725440176763459

I'll leave it up to you whether you want to merge this now and then update if needed once that conversation is resolved, or hold off.

**Comment by andrablaj** [comment]
Thanks @garethbowen! There is an open ticket https://github.com/medic/cht-tech-radar-implementers/issues/10 for documenting the way forward on merging those terms or not. The update can be done in a separate PR.

**Pull request merged**